### ISSUE TITLE
Untracked files not shown

### ISSUE THREAD

**Issue by rsignavong** [jonas/tig#1219]
Untracked files are `not shown` even with
```
set status-show-untracked-dirs	= yes		# Show files in untracked directories? (status)
set status-show-untracked-files	= yes		# Show untracked files?
```
in my `.tigrc` file

TIG version:
```
> tig --version
tig version 2.5.6
ncursesw version 6.3.20211021
readline version 8.1
```
OS:
MacOS Monterey 12.4
M1 Apple Chip

**Comment by koutcher** [comment]
2.5.6 fixed the reading of status.showuntrackedfiles from git config. Values from git config override the values in `.tigrc`.

**Comment by von** [comment]
So, I did confirm that with 2.5.6 having `showUntrackedFiles = true` in git config causes tig to behave as expected.

But As I read the [git-config documentation](https://git-scm.com/docs/git-config), the allowed values for `showUntrackedFiles` are `no/normal/alll` (not `true/false`) but tig seems to treat all those values as `false`.

It seems like tig should be treating `normal` or `all` for `showUntrackedFiles` as `true`.

**Comment by koutcher** [comment]
Agreed, not only the name needed to be fixed, the parsing as well.

**Comment by von** [comment]
Confirming fixed in 2.5.7
